### PR TITLE
Introduce `tofu-doobie-logging`, `examples` and improve doobie example

### LINKED ISSUE #703: Move examples to separate module
As was suggested in telegram chat it would be nice to move all examples to separate module(examples) and place it near module 'modules'.

### PR DESCRIPTION
**Pull request by FunFunFine**
fix #703

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -289,6 +289,15 @@ lazy val doobie  = project
   )
   .dependsOn(core, derivation, env % Test, zioInterop % Test)
 
+lazy val examples = project
+  .in(file("examples"))
+  .settings(
+    libraryDependencies ++= List(doobieCore, derevo, monix),
+    defaultSettings,
+    name := "tofu-examples",
```
Comment (Odomontois) [inline build.sbt:297]: examples should be skipped during publish, use noPublishSettinga

Comment (FunFunFine) [reply build.sbt:297]:
```suggestion
    name := "tofu-examples",
    noPublishSettings,
```

**Review by catostrophe — APPROVED** [review]

File: examples/src/main/resources/logback.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                    <onMatch>ACCEPT</onMatch>
+                </filter>
+                <filter class="ch.qos.logback.classic.filter.LevelFilter">
+                    <level>WARN</level>
+                    <onMatch>DENY</onMatch>
+                </filter>
+                <filter class="ch.qos.logback.classic.filter.LevelFilter">
+                    <level>ERROR</level>
+                    <onMatch>DENY</onMatch>
+                </filter>
```
Comment (FunFunFine) [inline examples/src/main/resources/logback.xml:29]: Why do we need all this stuff?

File: examples/src/main/scala/tofu/example/doobie/TofuDoobieExample.scala [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+}
+
+object DeptSql extends LoggingCompanion[DeptSql] {
+  def make[DB[_]: Monad: LiftConnectionIO: EmbeddableLogHandler]: DeptSql[DB] = {
+    EmbeddableLogHandler[DB].embedLift(implicit lh => new Impl)
+  }
+
+  final class Impl(implicit lh: LogHandler) extends DeptSql[ConnectionIO] {
+    def init: ConnectionIO[Unit]                   =
+      lsql"create table if not exists department(id int8, name varchar(50))".update.run.void
```
Comment (FunFunFine) [inline examples/src/main/scala/tofu/example/doobie/TofuDoobieExample.scala:75]: I feel like `logsql"..." `would be better here.

File: examples/src/main/scala/tofu/example/doobie/TofuDoobieExample.scala [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+import tofu.doobie.transactor.Txr
+import tofu.doobie.LiftConnectionIO
+import tofu.higherKind.derived.representableK
+import tofu.lift.{Lift, UnliftIO}
+import tofu.logging.derivation.{loggable, loggingMidTry}
+import tofu.logging.{Logging, LoggingCompanion, Logs}
+import tofu.syntax.context._
+import tofu.syntax.monadic._
+import tofu.syntax.doobie.log.handler._
+import tofu.syntax.doobie.log.string._
```
Comment (FunFunFine) [inline examples/src/main/scala/tofu/example/doobie/TofuDoobieExample.scala:21]: `import tofu.syntax.doobie.log._`
This looks like more unified way to import syntax

Comment (catostrophe) [reply examples/src/main/scala/tofu/example/doobie/TofuDoobieExample.scala:21]: These 2 imports should be usually put in different files.

File: examples/src/main/scala/tofu/example/doobie/TofuDoobieExample.scala (hunk 3 above)
Comment (catostrophe) [reply examples/src/main/scala/tofu/example/doobie/TofuDoobieExample.scala:75]: Let's make a poll in tofu chat.

File: examples/src/main/resources/logback.xml (hunk 2 above)
Comment (catostrophe) [reply examples/src/main/resources/logback.xml:29]: Dunno, just copied it from prod.

**Pull request merged**